handshake_client.go

```diff
@@ -600,6 +600,9 @@ func (hs *clientHandshakeState) doFullHandshake() error {
 		}
 	}
 
+	/* sessionHash does not include CertificateVerify */
+	sessionHash := hs.finishedHash.Sum()
```

Comment:
I know this file might not be strictly following this but, could you enclose your modified sections in any added file (file name NOT starting with `u_`) like below: 

```
// [UTLS SECTION START]
... modified section
// [UTLS SECTION END]
```

This will greatly help with readability and maintainability. Note we are routinely syncing with upstream and any modification leading to merge conflicts, without properly labeled as modified by us, might be overwritten/discarded.

Reply:
Or I can do it if you don't mind.

Reply:
I've added the the section comments. the whole of the extended-master-secret is a UTLS addition so I put the section comments around the whole if statement that checks for extended-master-secret.

Reply:
Thanks a ton!